bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Display.java

```diff
@@ -3008,6 +3008,13 @@ Point mapInPixels (Control from, Control to, int x, int y) {
 	return new Point (point.x, point.y);
 }
 
+private int getZoomLevelForMapping(Control from, Control to) {
+	if (to != null) {
+		return to.getZoom();
+	}
+	return from.getZoom();
```

Comment:
At other places, the `from` and `to` values are checked and throw report proper SWT error, such as 
```java
if (from != null && from.isDisposed()) error (SWT.ERROR_INVALID_ARGUMENT);
```
Should this be done here as well?

Reply:
Yeah right. it is used in map methods which uses mapInPixels, so it could have the same effect.